index.html

```diff
@@ -205,25 +193,25 @@ <h2>
         To <dfn>obtain permission</dfn>, run these steps <a>in parallel</a>:
       </p>
       <ol>
-        <li>
-          If there is a <a>prearranged trust relationship</a>,
-          return <code>true</code>.
-        </li>
         <li>
           Let <var>permission</var> be the result of running
           <a>query a permission</a> for the
           <a>wake lock permission name</a>.
         </li>
+        <li>
+          If <var>permission</var> is <code><a>"denied"</a></code>,
+          return <code>false</code>.
+        </li>
+        <li>
+          If there is a <a>prearranged trust relationship</a>,
```

Comment:
Moving this step after checking `permission == 'denied'` seems to contradict the statement above that if a prearranged trust relationship has been established then expressed permission from the user is not needed. Or perhaps it should be up to the implementer whether or not this kind of prearranged agreement can or cannot be overridden by the user either in the positive or negative direction.

Reply:
I changed the wording for now, so that it doesn't contradict.